models/issues/issue.go

```diff
@@ -341,6 +360,11 @@ func (issue *Issue) LoadAttributes(ctx context.Context) (err error) {
 	return issue.loadReactions(ctx)
 }
 
+// IsPinned returns if a Issue is pinned
+func (issue *Issue) IsPinned() bool {
+	return issue.PinOrder != 0
```

Comment:
It needs to add an panic check: if pinOrderLoaded=false, panic to tell developers that there is a bug.

Reply:
If you'd like to use the convention that `PinOrder != 0` means "loaded", you can use `PinOrder=-1` means loaded, then no need to introduce another fragile pinOrderLoaded field.

Reply:
6b651680088b7a167e0023659f2506b8632b571d